priv/templates/data_table.html.eex

```diff
@@ -0,0 +1,34 @@
+<h3>Data Table</h3>
+<table>
+  <thead>
+    <tr>
+      <th class="text-left">Name</th>
+      <th>Iterations per Second</th>
+      <th>Average</th>
+      <th>Deviation</th>
+      <th>median</th>
+      <th>minimum</th>
+      <th>maximum</th>
+      <th>Sample size</th>
+    </tr>
+  </thead>
+  <tbody>
+    <%= for {name, %{measurements: measurements, units: units}} <- statistics do %>
+      <tr>
+        <%= case Keyword.get(options, :path_information) do %>
+        <% {base_name, input_name} -> %>
+          <td class="text-left"><a href="<%= relative_file_path(base_name, [input_name, name]) %>"><%= name %></a></td>
+        <% nil -> %>
+          <td class="text-left"><%= name %></td>
+        <% end %>
+        <td><%= format_count measurements.ips, units.run_time %></td>
+        <td><%= format_duration measurements.average, units.run_time %></td>
+        <td><%= format_percent measurements.std_dev_ratio %></td>
+        <td><%= format_duration measurements.median, units.run_time %></td>
+        <td><%= format_duration measurements.minimum, units.run_time %></td>
+        <td><%= format_duration measurements.maximum, units.run_time %></td>
+        <td><%= measurements.sample_size %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
I think this one was comitted accidentally and we rather wanna keep using the one in partials, correct? :)

Reply:
correct